src/components/EditSections/EditReadingRoomAccess/EditReadingRoomAccess.js

```diff
@@ -47,13 +76,24 @@ const EditReadingRoomAccess = ({
       label={<Headline size="large" tag="h3"><FormattedMessage id="ui-users.readingRoom.readingRoomAccess" /></Headline>}
       displayWhenClosed={<Badge>{formData.length}</Badge>}
     >
+      <OnChange name="readingRoomsAccessList">
+        {(updatedRRAFormValues) => {
+          updateSortedRecordDetails(updatedRRAFormValues);
+        }}
+      </OnChange>
       <MultiColumnList
         striped
-        contentData={formData}
+        contentData={sortedRecordsDetails.data}
         columnMapping={columnMapping}
         visibleColumns={visibleColumns}
         formatter={getFormatter(form)}
         columnWidths={columnWidths}
+        sortOrder={sortedRecordsDetails.order}
+        sortDirection={`${sortedRecordsDetails.direction}ending`}
+        onHeaderClick={(e, meta) => setSortedRecordsDetails(
+          getReadingRoomSortedData(e, meta, sortedRecordsDetails)
```

Comment:
add semicolon add the end of the statement. Doesn't lint complain about this?

Reply:
It is not required here as getReadingRoomSortedData() will return the sortedRecords which will set the state of sortedRecordsDetails.